packages/aws-fargate/images/test-images/Dockerfile-alpine

```diff
@@ -1,8 +1,7 @@
-ARG DOCKER_TAG=latest
-ARG INSTANA_LAYER=icr.io/instana/aws-fargate-nodejs
+ARG INSTANA_LAYER=icr.io/instana/aws-fargate-nodejs:latest
```

Comment:
The INSTANA_LAYER env var now always contains a tag. Thus, the extra env var DOCKER_TAG is no longer needed.